app/assets/stylesheets/master.scss

```diff
@@ -236,4 +235,18 @@ p {
 
 .modal-backdrop {
   z-index: 1030;
-}
\ No newline at end of file
+}
+
+.dashboard{
+  background-color: #D1D1D1;
+  min-height: 200px;
+  padding: 30px;
+}
+.user-photo{
+  border-radius: 50%;
+  border: 5px solid #AAAAAA;
+  height: 150px;
+  width: 150px;
+  background:url(trump.jpg) center;
```

Comment:
I added this because I thought it would save us time to already have a  template to add code to, @NKHolton. None of the Gravatar stuff is done at all. We are doing that tomorrow! 👯